__tests__/app.test.js

```diff
@@ -156,4 +179,32 @@ describe('errors', () => {
             expect(body).toEqual({msg: 'no comments matching that id'})
         })
     })    
+    test('server returns 404 and not found msg when recieves post request to undefined endpoint', () => {
+        return request(app)
+        .post('/api/topiary')
+        .expect(404)
```

Comment:
This test has nothing to do with this endpoint and should already be handled by your express app.all('*') in app.js